posthog/api/organization_member.py

```diff
@@ -107,6 +113,7 @@ class OrganizationMemberViewSet(
             ),
             Prefetch("user__social_auth", queryset=UserSocialAuth.objects.all()),
         )
+        .annotate(last_login=F("user__last_login"))
```

Comment:
This `.annotate()` seems unused, as `get_last_login()` doesn't rely on it

Reply:
we need the `.annotate()`; without it, the last_login property is not a part of the response sent to the FE.
I removed the `get_last_login()` method instead. It works as expected now. Please check